atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceIntegrationTest.java

```diff
@@ -137,26 +139,27 @@ public void testCreateTableCaseInsensitive() throws TException {
 
     @Test
     public void testGcGraceSecondsUpgradeIsApplied() throws TException {
-        CassandraKeyValueService kvs = createKvs(getConfigWithGcGraceSeconds(FOUR_DAYS_IN_SECONDS));
+        Logger testLogger = mock(Logger.class);
+        CassandraKeyValueService kvs = createKvs(getConfigWithGcGraceSeconds(FOUR_DAYS_IN_SECONDS), testLogger);
         //first startup same as initial - no upgrade
-        verify(logger, times(1))
-                .debug(startsWith("No tables are being upgraded on startup. No updated table-related settings found."));
+        verify(testLogger, times(1))
+                .info(startsWith("No tables are being upgraded on startup. No updated table-related settings found."));
         kvs.createTable(testTable, AtlasDbConstants.GENERIC_TABLE_METADATA);
         assertThatGcGraceSecondsIs(kvs, FOUR_DAYS_IN_SECONDS);
         kvs.close();
 
-        CassandraKeyValueService kvs2 = createKvs(getConfigWithGcGraceSeconds(ONE_HOUR_IN_SECONDS));
+        CassandraKeyValueService kvs2 = createKvs(getConfigWithGcGraceSeconds(ONE_HOUR_IN_SECONDS), testLogger);
         assertThatGcGraceSecondsIs(kvs2, ONE_HOUR_IN_SECONDS);
-        kvs.close();
+        kvs2.close();
         //startup with different GC grace seconds - should upgrade
-        verify(logger, times(1))
-                .debug(startsWith("New table-related settings were applied on startup!!"));
+        verify(testLogger, times(1))
+                .info(startsWith("New table-related settings were applied on startup!!"));
 
-        CassandraKeyValueService kvs3 = createKvs(getConfigWithGcGraceSeconds(ONE_HOUR_IN_SECONDS));
+        CassandraKeyValueService kvs3 = createKvs(getConfigWithGcGraceSeconds(ONE_HOUR_IN_SECONDS), testLogger);
         assertThatGcGraceSecondsIs(kvs3, ONE_HOUR_IN_SECONDS);
         //startup with same gc grace seconds as previous one - no upgrade
-        verify(logger, times(2))
-                .debug(startsWith("No tables are being upgraded on startup. No updated table-related settings found."));
+        verify(testLogger, times(2))
+                .info(startsWith("No tables are being upgraded on startup. No updated table-related settings found."));
         kvs3.close();
```

Comment:
Yep, this is nice! Also looks cleaner than what I initially thought of (a `Mockito.reset(logger)` at the beginning of this test)